### ISSUE TITLE
user/526: /usr/bin/telnet segfaults on execution

### ISSUE THREAD

**Issue by ligurio** [ligurio/testrepo#66]

**Comment by ligurio** [comment]
```
From:    Theo de Raadt <email@00000423>
Date:    1998-07-16 19:00:06
Subject: Re: user/526: /usr/bin/telnet segfaults on execution
Thread:  90224200300023

The following reply was made to PR user/526; it has been noted by GNATS.

From: Theo de Raadt <email@00000423>
To: email@00008695
Cc: email@00000006
Subject: Re: user/526: /usr/bin/telnet segfaults on execution 
Date: Thu, 16 Jul 1998 12:53:07 -0600

 Not repeatable:
 
 [i386 deraadt 1 ]> telnet
 telnet> quit
 
 Come on -- you can all give MUCH better bug reports than that!  RUN
 THE DEBUGGER AGAINST THE PROGRAM!  PROVIDE A TRACEBACK!  TAKE A SHOT AT ACTUALLY
 FIGURING OUT WHAT IS GOING WRONG!  Did you even read the /etc/motd on your
 system?  It says:
 
 > Please use the sendbug(1) utility to report bugs in the system.
 > Before reporting a bug, please try to reproduce it with the latest
 > version of the code.  With bug reports, please try to ensure that
 > enough information to reproduce the problem is enclosed, and if a
 > known fix for it exists, include that as well.
 
 Please!
 


```

**Comment by ligurio** [comment]
```
From:    Matthieu Herrb <email@00003425>
Date:    1998-07-16 21:17:10
Subject: Re: user/526: /usr/bin/telnet segfaults on execution
Thread:  90224200300023

You wrote (in your message from Thu 16)
 > Subject: Re: user/526: /usr/bin/telnet segfaults on execution 
 > Date: Thu, 16 Jul 1998 12:53:07 -0600
 > 
 >  Not repeatable:
 >  
 >  [i386 deraadt 1 ]> telnet
 >  telnet> quit
 >  

I don't want to defend users that don't give enough information in
their PR's but I could repeat it easyly with rev 1.18 of 
/usr/src/usr.bin/telnet/commands.c.

Since it was  trashing the stack by writing one byte to far in
hbuf[MAXHOSTNAMELEN] , it was a little difficult to get useful 
information out of gdb...

I was going to provide a fix but you fixed it (version 1.19) just 2mn
before replying that is was not reproductible.


					Matthieu


```
